### ISSUE TITLE
The Color Contrast ratio of keyboard focus indicator for the interactive controls on the video section of the page is "1.534:1" which is less than 3:1: A11y_TypeSpec_Video_Non-text Contrast

### ISSUE THREAD

**Issue by kupatkar99** [microsoft/typespec#8561]
### GIT Hub Tags:
#A11yTCS; #A11ySev2; #TypeSpec; #GH_TypeSpec_Web_Sept2025; #DesktopWeb; #A11yMAS; #WCAG1.4.11; #Non-Text Contrast; #AILimited; #Win11; #FTP; #ChromiumEdge;

### Environment and OS details:
Application name: typespec.io
URL: https://[typespec.io](https://typespec.io/)/
ChromiumEdge Version: 140.0.3485.66 (Official build) (64-bit)

### Reproduction Steps:

1. Open url "https://[typespec.io](https://typespec.io/videos/)/videos/" and login with valid Credentials.
2. "TypeSpec Videos" Page will get open.
3. Tab till Video links and share button.
4. Observe the issue whether Use is able to invoke the "Error" control via keyboard keys or not.

### Actual behavior:
The Color Contrast ratio of keyboard focus indicator for the interactive controls on the video section of the page is "1.534:1" which is less than 3:1

### Expected behavior:
The focus indicator for the interactive controls in the video section needs to have a contrast ratio of at least 3:1 compared to adjacent colors, so that users with visual impairments can clearly see it.

### User Impact:
Visual impairments users may not be able to identify the current focus location, making keyboard navigation difficult and reducing overall usability.

### Attachment:

<img width="1526" height="818" alt="Image" src="https://github.com/user-attachments/assets/b38b77de-9575-42a3-bb74-88b204440172" />

**Comment by lfraleigh** [comment]
Since we are using a standard embed widget provided by YouTube, I believe we have no control over this.  Can someone else confirm?

**Comment by timotheeguerin** [comment]
yeah that one should be out of our control we just render an embedded iframe of youtube preview
